system-test/bigquery.ts

```diff
@@ -888,9 +887,25 @@ describe('BigQuery', () => {
       const description = 'catsandstuff';
       await table.setMetadata({description});
       const [metadata] = await table.getMetadata();
+      const metadataProps = Object.values(metadata);
+      assert.strictEqual(metadataProps.length, 18);
```

Comment:
I think checking for the number of attributes here can lead the test to be flaky in the long term, as every time that the service team adds a new field, the test is going to break.

Reply:
Good point!